cypress/support/pages/home.pageObject.js

```diff
@@ -1,10 +1,19 @@
 import PageObject from '../PageObject';
 
 class homePageObject extends PageObject {
-  url = '/#/';
+  url = '/';
 
   get usernameLink() {
```

Comment:
The data-cy attribute value should be more descriptive, consider changing '/profile' to 'profile-link'